src/main/java/br/unicamp/cst/bindings/soar/SOARPlugin.java

```diff
@@ -229,7 +181,7 @@ public void stopSOAR() {
         getAgent().stop();
     }
 
-    public void runSOAR() {
+    /*public void runSOAR() {
```

Comment:
Would be better to remove any commented code.

Reply:
Done